src/lang/Messages.properties

```diff
@@ -854,6 +854,7 @@ context.api.action.excludeAllContextTechnologies = Excludes all built in technol
 copy.copy.popup = Copy
 copy.desc       = Provides a right click option to copy highlighted text to the clipboard
 
+core.api.action.deleteSiteNode = Deletes the site node found in the Sites Tree on the basis of the URL and the Method passed. 
```

Comment:
I'd suggest "...of the URL, HTTP method, and post data (if applicable and specified)."